CorsixTH/Lua/app.lua

```diff
@@ -616,10 +631,9 @@ end
 --!return (string, error) Returns the found absolute path, or nil if not found. Then
 --!       a second variable is returned with an error message.
 function App:getAbsolutePathToLevelFile(level)
-  local path = debug.getinfo(1, "S").source:sub(2, -12)
   -- First look in Campaigns. If not found there, fall back to Levels.
-  local list_of_possible_paths = { self.user_level_dir, path .. "Campaigns", self.level_dir }
-  for _, parent_path in ipairs(list_of_possible_paths) do
+  local paths_to_search = { self.user_campaign_dir, self.campaign_dir, self.user_level_dir, self.level_dir }
```

Comment:
```suggestion
  local paths_to_search = {
    self.user_campaign_dir,
    self.campaign_dir,
    self.user_level_dir,
    self.level_dir
  }
```
Maybe too many lines?

Reply:
I considered that for about 2 seconds too, but it's not our code style.

Also, if you don't see it when you have 1 word at each line, would you see it if you have multiple words at each line?
I think the benefits are doubtful, and not enough to break code style.